src/snowflake/cli/_plugins/nativeapp/entities/application_package.py

```diff
@@ -220,6 +221,7 @@ def action_deploy(
             force=force,
         )
 
+    @span("drop_app_package")
```

Comment:
i thought we said we didn't want spans to be implementation-specific? shouldn't this be just "drop"? same for the others here

Reply:
i guess it depends what we want to see when we're looking at the metrics. i was under the impression that we'd have generic span names like "bundle", "deploy", "drop", etc so that we could aggregate metrics from different entity types together, but if we want to keep them separate then this name is ok